### PR TITLE
Updated Set-MpPreference cmdlet - task - 6669854

### PR DESCRIPTION
**Pull request by Ashok-Lobo**

### CODE REVIEW

File: docset/winserver2019-ps/defender/Set-MpPreference.md [hunk 1]
```diff
@@ -1012,6 +1010,18 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -ScanScheduleTime
+Specify the time of day to run a scheduled scan. The time refers to the local time on the computer. Specify the number of minutes after midnight (for example, enter 60 for 1 a.m.). That parameter has a default time of two hours after midnight (2:00 a.m.).
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/Set-MpPreference.md:1014]:
```suggestion
Specifies the time of day to run a scheduled scan. The time refers to the local time on the computer. Specify the number of minutes after midnight (for example, enter 60 for 1 a.m.). This parameter has a default time of two hours after midnight (2 a.m.).
```

File: docset/winserver2022-ps/defender/Set-MpPreference.md [hunk 2]
```diff
@@ -1415,6 +1414,18 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -ScanScheduleTime
+Specify the time of day to run a scheduled scan. The time refers to the local time on the computer. Specify the number of minutes after midnight (for example, enter 60 for 1 a.m.). That parameter has a default time of two hours after midnight (2:00 a.m.).
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/defender/Set-MpPreference.md:1418]:
```suggestion
Specifies the time of day to run a scheduled scan. The time refers to the local time on the computer. Specify the number of minutes after midnight (for example, enter 60 for 1 a.m.). This parameter has a default time of two hours after midnight (2 a.m.).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Ashok-Lobo, kindly commit the suggested changes. Many thanks.

**Comment by Ashok-Lobo** [comment]
@JohanFreelancer9 

I have committed the suggested changes. Thank you!

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by denisebmsft — APPROVED** [review]
Thank you @Ashok-Lobo @JohanFreelancer9

**Pull request merged**